up42/storage.py

```diff
@@ -223,37 +216,28 @@ def get_assets(
-                "2022-01-30".
-            geometry: Filter for assets intersecting the Polygon geometry. One of FeatureCollection, Feature,
-                dict (geojson geometry), list (bounds coordinates), GeoDataFrame, shapely.Polygon, shapely.Point.
-                All assume EPSG 4326!
-            workspace_id: Filter for assets by the workspace ID. You can use `storage.workspace_id` here
-                to limit to your own workspace.
-            collection_names: Filter for assets with any of the provided collection names, e.g. ["spot", "phr"].
-            producer_names: Filter for assets with any of the provided producer names, e.g. ["airbus", "21at"].
-            tags: Filter for assets with any of the provided tags, e.g. ["optical", "US"].
-            sources: Filter for assets with any of the provided sources, one or multiple of
-                ["ARCHIVE", "TASKING", "ANALYTICSPLATFORM", "USER"].
-            search: Filter for assets that contain the provided search query in their name, title, or order ID, e.g.
-                "SPOT 6/7 NY Central Park".
-            custom_filter: A CQL2 filter expression for filtering based on properties, see
-                https://pystac-client.readthedocs.io/en/stable/tutorials/cql2-filter.html#CQL2-Filters
-            limit: Optional, only return n first assets (by sorting and order criteria). Optimal to use if your
-                workspace contains many assets.
-            sortby: The sorting criteria, corresponds to the asset properties, e.g. "created_after".
-            descending: The sorting order, True for descending (default), False for ascending.
-            return_json: If set to True, returns json dict instead.
+            created_after: Search for assets created after the specified timestamp, in `"YYYY-MM-DD"` format.
+            created_before: Search for assets created before the specified timestamp, in `"YYYY-MM-DD"` format.
+            acquired_after: Search for assets that contain data acquired after the specified timestamp, in `"YYYY-MM-DD"` format.
+            acquired_before: Search for assets that contain data acquired before the specified timestamp, in `"YYYY-MM-DD"` format.
+            geometry: Search for assets that contain STAC items intersecting the provided geometry, in EPSG:4326 (WGS84) format.<br/>For more information on STAC items, see [Introduction to STAC](https://docs.up42.com/developers/api-assets/stac-about).
+            workspace_id: Search by the workspace ID.
+            collection_names: Search for assets from any of the provided geospatial collections.
+            producer_names: Search for assets from any of the provided producers.
+            tags: Search for assets with any of the provided tags.
+            sources: Search for assets from any of the provided sources. The allowed values: `"ARCHIVE"`, `"TASKING"`, `"ANALYTICSPLATFORM"`, `"USER"`.
```

Comment:
The `ANALYTICSPLATFORM` in API is `ANALYTICS`

Reply:
You are totally right, actually this also triggered a bug in the get_assets by source. (I have fixed that as well) Thanks.